packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/drp/configuration/common/mainPcc/dedicatedCloud-datacenter-drp-mainPccStep.html

```diff
@@ -23,13 +23,13 @@
         <oui-message data-type="warning">
             <span
                 data-translate="dedicatedCloud_datacenter_order_ip"
-                data-translate-values="{ 'sref': 'app.dedicatedClouds.datacenter.drp.ovh.mainPccStep.legacyOrderIp'}"
+                data-translate-values="{ 'sref': $ctrl.orderIpStateRef.legacy }"
```

Comment:
We should avoid ui-router references in our translations I think. Could this be turned to a `href` ? We'll still have html in the translations but no ui-router dependency